source/catalog.html

```diff
@@ -5,148 +5,145 @@
-            </ul>
-            <ul class="main-navigation__list shopping-list">
-              <li class="shopping-list__item">
-                <a class="shopping-list__link" href="#">
-                  Новые поступления
-                </a>
-              </li>
-              <li class="shopping-list__item">
-                <a class="shopping-list__link" href="#">
-                  Распродажа
-                </a>
-              </li>
-            </ul>
-            <ul class="main-navigation__list user-list">
-              <li class="user-list__item">
-                <a class="user-list__link" href="#">
-                  Поиск по сайту
-                </a>
-              </li>
-              <li class="user-list__item">
-                <a class="user-list__link" href="#">
-                  Корзина: пока пуста
-                </a>
-              </li>
-            </ul>
-          </nav>
-        </div>
+    <header class="page-header container">
+      <a class="page-header__logo logo" href="index.html">
+        <img src="img/mishka-logo-mobile.svg" width="86" height="35" alt="Логотип магазина мягких игрушек: Мишка">
+      </a>
+      <div class="page-header__menu">
+        <button class="page-header__burger page-header__burger--closed" type="button">
+          <span class="visually-hidden">Открыть меню</span>
+        </button>
+        <nav class="main-navigation">
+          <ul class="main-navigation__list site-list">
+            <li class="site-list__item">
+              <a class="site-list__link" href="catalog.html">
+                Каталог товаров
+              </a>
+            </li>
+            <li class="site-list__item">
+              <a class="site-list__link" href="form.html">
+                Вязание на заказ
+              </a>
+            </li>
+          </ul>
+          <ul class="main-navigation__list shopping-list">
+            <li class="shopping-list__item">
+              <a class="shopping-list__link" href="#">
+                Новые поступления
+              </a>
+            </li>
+            <li class="shopping-list__item">
+              <a class="shopping-list__link" href="#">
+                Распродажа
+              </a>
+            </li>
+          </ul>
+          <ul class="main-navigation__list user-list">
+            <li class="user-list__item">
+              <a class="user-list__link" href="#">
+                Поиск по сайту
+              </a>
+            </li>
+            <li class="user-list__item">
+              <a class="user-list__link" href="#">
+                Корзина: пока пуста
+              </a>
+            </li>
+          </ul>
+        </nav>
       </div>
     </header>
     <main class="page-main">
-      <section class="intro">
-        <div class="intro__container container">
-          <h1 class="intro__title">Каталог товаров</h1>
-        </div>
+      <section class="intro container">
+        <h1 class="intro__title">Каталог товаров</h1>
```

Comment:
Я имел ввиду на заголовок:
```suggestion
      <section class="intro">
        <h1 class="intro__title container">Каталог товаров</h1>
```

Контейнер же должен быть внутри блока. На самом блоке он может быть только если у него фон не тянется (Но вполне допустимо для единообразия сделать контейнеры всегда внутри)